### ISSUE TITLE
django 1.11 and 2.0 support

### ISSUE THREAD

**Issue by auvipy** [nitmir/django-cas-server#33]

**Comment by nitmir** [comment]
Did you test it ?

**Comment by godlike64** [comment]
I just tried to run django-cas-server using django-2.0. It fails (another CAS server package which has no django-2.0 compatibility has the same issue):

```
$ ./manage.py migrate
Traceback (most recent call last):
  File "./manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/home/godlike/virtualenv/36/lib/python3.6/site-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "/home/godlike/virtualenv/36/lib/python3.6/site-packages/django/core/management/__init__.py", line 347, in execute
    django.setup()
  File "/home/godlike/virtualenv/36/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/godlike/virtualenv/36/lib/python3.6/site-packages/django/apps/registry.py", line 112, in populate
    app_config.import_models()
  File "/home/godlike/virtualenv/36/lib/python3.6/site-packages/django/apps/config.py", line 198, in import_models
    self.models_module = import_module(models_module_name)
  File "/home/godlike/virtualenv/36/lib64/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/godlike/projects/cas-testbed/casserver2/cas_server/models.py", line 31, in <module>
    from cas_server import utils
  File "/home/godlike/projects/cas-testbed/casserver2/cas_server/utils.py", line 15, in <module>
    from django.core.urlresolvers import reverse
ModuleNotFoundError: No module named 'django.core.urlresolvers'
```

**Comment by nitmir** [comment]
This should be fixed by 4123450e9f9bdd1ca965f224f4ce41716d10654f.
The test suite is known running with success for Django 2.0
